src/components/AbstraxionSignin/index.tsx

```diff
@@ -97,6 +99,17 @@ export const AbstraxionSignin = () => {
     suggestAndConnect({ chainInfo: testnetChainInfo, walletType: wallet });
   };
 
+  async function handleWebauthnAuthenticate() {
+    try {
+      await stytchClient.webauthn.authenticate({
+        domain: "localhost",
```

Comment:
This is fine for now but we should probably start figuring out how to (and stay with me here) _abstract_ a lot of these config values into environment variables.